### PR TITLE
#21: adding settings and configurations to send the message to anyone

### PR DESCRIPTION
**Pull request by sharayu0**

### CODE REVIEW

**Comment by sharayu0** [comment]
Message sent from one user to another or between active user is done properly.
Also messages are rearranged in the required order.
set to display how the display the user as entity id, label, as a rendered entity or author and set the configurations around it.

**Comment by sharayu0** [comment]
was trying to use the view to display private message thread of the member users but not worked for the user to display recepient user to which current user sending message.
worked on Customized private message threads, Customized private messages , Customized the display of thread members.

File: web/sites/default/settings.php [hunk 1]
```diff
@@ -283,7 +283,7 @@
  * @endcode
  */
 $settings['hash_salt'] =
-    'xkjKO_vt9B4y64ikdGrp516v44dl51XvaTsVH2GShCUdvISPYvLROuXbsTpng5f1BwmLTjCVSQ';
+    '8IgDpLP4DIfk_J3udJe68AMGdUukbgOo5RpnuE1KIQQi-jQLGT5D2CbXZGnUY6PvA3yV10Qs8Q';
```
Comment (JayKandari) [inline web/sites/default/settings.php]: Is this change required ? if not can we remove this.

**Review by JayKandari — CHANGES_REQUESTED** [review]

**Review by JatinGupta40 — APPROVED** [review]

**Review by JayKandari — APPROVED** [review]
LGTM.

**Pull request merged**